pkg/store/tsdb.go

```diff
@@ -211,7 +212,7 @@ func (s *TSDBStore) translateAndExtendLabels(m, extend labels.Labels) []storepb.
 }
 
 // LabelNames returns all known label names.
-func (s *TSDBStore) LabelNames(ctx context.Context, _ *storepb.LabelNamesRequest) (
+func (s *TSDBStore) LabelNames(_ context.Context, _ *storepb.LabelNamesRequest) (
 	*storepb.LabelNamesResponse, error,
 ) {
 	q, err := s.db.Querier(context.Background(), math.MinInt64, math.MaxInt64)
```

Comment:
```suggestion
	q, err := s.db.Querier(ctx, math.MinInt64, math.MaxInt64)
```
?

Reply:
I'm not sure why the given context was ignored previously. Assuming there's a reason for it I haven't changed anything regarding that.

I just wanted to make it explicit that the given context is not used. We should address this context propagation issue in another PR.

Reply:
Never mind, fixed it.